examples/java/src/main/java/org/apache/beam/examples/complete/TopWikipediaSessions.java

```diff
@@ -167,7 +167,8 @@ public ComputeTopSessions(double samplingThreshold) {
               new DoFn<String, String>() {
                 @ProcessElement
                 public void processElement(ProcessContext c) {
-                  if (Math.abs(c.element().hashCode()) <= Integer.MAX_VALUE * samplingThreshold) {
+                  long hashCode = c.element().hashCode();
```

Comment:
Yes unfortunately firebug doesn't like the typecasting of hashCode which returns an int and hence i have assigned it to a long and then used that long in further checks as you highlighted in the next line.